brunch-config.js

```diff
@@ -31,8 +31,7 @@ exports.config = {
   plugins: {
     babel: {
       // Do not use ES6 compiler in vendor code
-      ignore: [/^(web\/static\/vendor)/],
-      loose: "all"
```

Comment:
It was added for IE8 and IE9 support iirc. If we remove it we should
definitely test IE again.

On Thursday, 26 November 2015, Chris McCord <notifications@github.com>
wrote:

> In brunch-config.js
> <https://github.com/phoenixframework/phoenix/pull/1374#discussion_r45997822>
> :
>
> > @@ -31,8 +31,7 @@ exports.config = {
> >    plugins: {
> >      babel: {
> >        // Do not use ES6 compiler in vendor code
> > -      ignore: [/^(web\/static\/vendor)/],
> > -      loose: "all"
>
> @ericmj <https://github.com/ericmj> do you remember why we added loose?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/phoenixframework/phoenix/pull/1374/files#r45997822>.
>


-- 
Eric Meadows-Jönsson